CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 6.4.4
```

Comment:
This will be conflict with my recent [PR](https://github.com/logstash-plugins/logstash-input-beats/pull/464) but I will sort it out if you merge first.